<launch>
    <arg name="tbName" default="asd"/>
    <arg name="x" default="0.0"/>
    <arg name="y" default="0.0"/>

    <group ns="$(arg tbName)">
    <param name="tf_prefix" value="$(arg tbName)" />
    <include file="$(find multi_robot)/launch/TD_one_robot.launch" >
      <arg name="tbName" value="$(arg tbName)"/>
      <arg name="x_pos" value="$(arg x)"/>
      <arg name="y_pos" value="$(arg y)"/>
      <arg name="z_pos" value="0"/>
    </include> 
    </group>



  <include file="$(find multi_robot)/launch/TD_tb3_move_base.launch">
    <arg name="tbName" value="$(arg tbName)"/>
  </include>
  <!-- <include file="$(find spqrel_navigation)/ros/launch/spqrel_navigation.launch">
  </include> -->

  <group ns="$(arg tbName)">
    <node name="roboNode" pkg="multi_robot" type="roboNode.py" output="screen" clear_params="true">
      <param name="mode" value="$(arg tbName)" /> 
    </node>
  </group>

</launch>